Sources/docc/DocCDocumentation.docc/adding-structure-to-your-documentation-pages.md

```diff
@@ -150,7 +150,7 @@ with the symbol path of the symbol you're organizing and rename the file accordi
 # ``SlothCreator/Sloth``
 ```
 
-> Important: The symbol path for the page title of an extension file need to start
+> Important: The symbol path for the page title of an extension file needs to start
```

Comment:
I'm a non-native speaker so apologies if this is incorrect. 

My understanding is that "need" is used for singular subjects and "needs" for plural subjects. 

In this sentence the symbol path is the subject. I read this sentence as referring to a single extension file which has a single symbol path. As such, my understanding is that "need" is grammatically correct here.

Reply:
In this case, the verb "to need" is irregular; the third-person singular present form with a gerund-participle (as in the usage here) is "needs". Ref.: https://en.wiktionary.org/wiki/need#:~:text=gerund-participle